gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -50,6 +50,7 @@
             "LambdaMethodReference",
             "LoggerEnclosingClass",
             "LogsafeArgName",
+            "LogsafeArgument",
```

Comment:
It's only necessary to update the extension with names of checks that are automatically fixed by default, no need to register this one since it doesn't have an automatic-fix.
```suggestion
```